src/schemas/CohortMembershipSchema.js

```diff
@@ -15,10 +15,25 @@ module.exports = (conn) => {
       enum: ['student', 'coach', 'instructor', 'admin', 'hr'],
       required: true,
     },
-    state: {
+    isActive: {
```

Comment:
```suggestion
    active: {
```

Hemos venido usando adjetivos y sustantivos sin ningun problema para poder entender a que hacen referencia, es más, es común el uso de adjetivos en booleanos